src/app/Scenes/PartnerOffer/PartnerOfferContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import { Flex } from "@artsy/palette-mobile"
+import { usePartnerOfferCheckoutMutation$data } from "__generated__/usePartnerOfferCheckoutMutation.graphql"
+import { LoadingSpinner } from "app/Components/Modals/LoadingModal"
+import { usePartnerOfferMutation } from "app/Scenes/PartnerOffer/mutations/usePartnerOfferCheckoutMutation"
+import { goBack, navigate } from "app/system/navigation/navigate"
+import { useEffect } from "react"
+
+export const PartnerOfferContainer: React.FC<{ partnerOfferID: string }> = ({ partnerOfferID }) => {
+  const handleRedirect = (response: usePartnerOfferCheckoutMutation$data) => {
+    const orderOrError = response.commerceCreatePartnerOfferOrder?.orderOrError
+
+    if (orderOrError?.error) {
+      const { code, data } = orderOrError.error
+      const artworkId = JSON.parse(data?.toString() ?? "")?.artwork_id
```

Comment:
Since `JSON.parse("")` will throw an exception, I guess you probably want something like ``
```suggestion
      const artworkId = JSON.parse(data?.toString() ?? "{}")?.artwork_id
```